tests/integration/modules/git.py

```diff
@@ -581,10 +579,13 @@ def test_init(self):
         '''
         Use git.init to init a new repo
         '''
-        new_repo = tempfile.mkdtemp(dir=integration.TMP)
+        new_repo = os.path.normcase(tempfile.mkdtemp(dir=integration.TMP))
+        if salt.utils.is_windows():
+            new_repo = new_repo.replace('\\', '/')
+
         self.assertEqual(
-            self.run_function('git.init', [new_repo]),
-            'Initialized empty Git repository in {0}/.git/'.format(new_repo)
+            self.run_function('git.init', [new_repo]).lower(),
+            'initialized empty git repository in {0}/.git/'.format(new_repo)
```

Comment:
This is going to produce inconsistent results. You're going to get mixed case for new_repo on some platforms and then compare it to something that's forced to lower-case.